HOWTOCONTRIBUTE.rst

```diff
@@ -160,6 +160,39 @@ Preparing Pull Requests
 
    This command will run tests via the "pytest" tool against Python 3.6.
 
+#. Running the performance test suite
+
+Performance matters and it is worth considering whether your code has introduced
+performance regressions. *climpred* is starting to write a suite of benchmarking tests
+using `asv <https://asv.readthedocs.io/en/stable/>`__
+to enable easy monitoring of the performance of critical *climpred* operations.
+These benchmarks are all found in the ``climpred/asv_bench`` directory.
+
+If you need to run a benchmark, change your directory to ``asv_bench/`` and run::
+
+    $ asv continuous -f 1.1 upstream/master HEAD
+
+You can replace ``HEAD`` with the name of the branch you are working on,
+and report benchmarks that changed by more than 10%.
```

Comment:
Maybe add this to the `PR_TEMPLATE` code that says that you've run `asv` and have reported any change +10%.